pkg/distributors/test/DistributionScheduler.test.ts

```diff
@@ -0,0 +1,209 @@
+import { ethers } from 'hardhat';
+import { Contract, BigNumber, BigNumberish } from 'ethers';
+import { expect } from 'chai';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
+
+import Token from '@balancer-labs/v2-helpers/src/models/tokens/Token';
+import TokenList from '@balancer-labs/v2-helpers/src/models/tokens/TokenList';
+
+import { fp } from '@balancer-labs/v2-helpers/src/numbers';
+
+import { deploy } from '@balancer-labs/v2-helpers/src/contract';
+import { expectBalanceChange } from '@balancer-labs/v2-helpers/src/test/tokenBalance';
+import * as expectEvent from '@balancer-labs/v2-helpers/src/test/expectEvent';
+import { advanceTime, currentTimestamp } from '@balancer-labs/v2-helpers/src/time';
+import { ZERO_BYTES32 } from '@balancer-labs/v2-helpers/src/constants';
+import { MultiDistributor } from './helpers/MultiDistributor';
+import TypesConverter from '@balancer-labs/v2-helpers/src/models/types/TypesConverter';
+import { solidityKeccak256 } from 'ethers/lib/utils';
+import { Account } from '@balancer-labs/v2-helpers/src/models/types/types';
+import Vault from '@balancer-labs/v2-helpers/src/models/vault/Vault';
+
+describe('Distribution Scheduler', () => {
+  let vault: Vault;
+  let distributor: MultiDistributor;
+  let scheduler: Contract;
+
+  let stakingToken: Token, stakingTokens: TokenList;
+  let distributionToken: Token, distributionTokens: TokenList;
+
+  let distributionOwner: SignerWithAddress, poker: SignerWithAddress;
+
+  before('setup signers', async () => {
+    [, distributionOwner, poker] = await ethers.getSigners();
```

Comment:
```suggestion
    [, distributionOwner, other] = await ethers.getSigners();
```

The 'poker' bit had me do a double take :joy: